distribution/scenes/textures/patterns/crackle_solid.pov

```diff
@@ -2,97 +2,190 @@
+    normal {bumps .002 scale <.1,.1,10> rotate y*90}
 }
 
 #declare Zinc3 = texture {
- pigment {Gray60}
- finish {phong .5 reflection {.3}}
- normal {bumps .002 scale <.1,.1,10> rotate y*180}
+    pigment {Gray60}
+    finish {phong .5 reflection {.3}}
+    normal {bumps .002 scale <.1,.1,10> rotate y*180}
 }
 
 #declare Zinc = texture {
- crackle solid
- texture_map {
-  [0.0 Zinc1]
-  [0.5 Zinc2]
-  [1.0 Zinc3]
- }
- scale 1
+    crackle solid
+    texture_map {
+        [0.0 Zinc1]
+        [0.5 Zinc2]
+        [1.0 Zinc3]
+    }
 }
 
 //--floor--------------------------------
-plane {y,0
-       texture {checker texture {Stone1} texture {Stone2} scale 20 translate y*10}
-       }
+plane {
+    y,0
+    texture {
+        checker
+        texture {Stone1}
+        texture {Stone2}
+        scale 20
+        translate y*10
+    }
+}
+
+//--Wood Texture (A 3.7 version of T_Wood21)
+#declare ColorWood11A_00 = srgb <0.80392,0.67451,0.239220>;
+#declare ColorWood11A_01 = srgb <0.60392,0.33726,0.011765>;
+#declare ColorWood11A_02 = srgb <0.80392,0.67451,0.239220>;
+#declare ColorWood11A_03 = srgb <0.53333,0.28235,0.003922>;
+#declare ColorWood11A_04 = srgb <0.80392,0.67451,0.239220>;
+#declare M_Wood11A = color_map {
+    blend_mode 2 blend_gamma 2.5
+    [0.000, 0.222 color ColorWood11A_00
+                  color ColorWood11A_00]
+    [0.222, 0.342 color ColorWood11A_00
+                  color ColorWood11A_01]
+    [0.342, 0.393 color ColorWood11A_01
+                  color ColorWood11A_02]
+    [0.393, 0.709 color ColorWood11A_02
+                  color ColorWood11A_02]
+    [0.709, 0.821 color ColorWood11A_02
+                  color ColorWood11A_03]
+    [0.821, 1.000 color ColorWood11A_03
+                  color ColorWood11A_04]
+}
+#declare ColorWood11B_Clr = srgbt <1.00,1.00,1.00,1.00>;
+#declare ColorWood11B_00  = srgbt <0.70588,0.41176,0.086275,0.60>;
+#declare ColorWood11B_01  = srgbt <0.70588,0.46275,0.086275,0.60>;
+#declare M_Wood11B = color_map {
+    blend_mode 2 blend_gamma 2.5
+    [0.000, 0.120 color ColorWood11B_Clr
+                  color ColorWood11B_00]
+    [0.120, 0.231 color ColorWood11B_00
+                  color ColorWood11B_01]
+    [0.231, 0.496 color ColorWood11B_01
+                  color ColorWood11B_Clr]
+    [0.496, 0.701 color ColorWood11B_Clr
+                  color ColorWood11B_Clr]
+    [0.701, 0.829 color ColorWood11B_Clr
+                  color ColorWood11B_01]
+    [0.829, 1.000 color ColorWood11B_01
+                  color ColorWood11B_Clr]
+}
+#declare P_WoodGrain1A = pigment {
+    wood
+    scale <0.05,.05,1>
+    warp { turbulence 0.04 octaves 3 }
+}
+#declare P_WoodGrain1B = pigment {
+    wood
+    scale <0.15,.5,1>
+    warp { turbulence <0.1,0.5,1> octaves 5 lambda 3.25 }
+    rotate <5,10,5>
+    translate -x*2
+}
+#declare T_Wood21 =
+    texture { pigment{ P_WoodGrain1A color_map { M_Wood11A }}}
+    texture { pigment{ P_WoodGrain1B color_map { M_Wood11B }}}
```

Comment:
Please see my comment to the colors.inc decoupling for explicit srgb encoding for the bulk of my reasoning. 

I'll add for all the woods.inc we have the issue these overlap textures with transparency - which as I highlighted in a gamma related post tp the newsgroup doesn't work as 'tightly' in an assumed_gamma '1' space. I implemented something for assumed_gamma 1, but I don't like it as an equivalent replacement. The local coding is better, but a match requires more than the map I did.